perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -974,3 +975,225 @@ def MoveFile(self, target, source_path, remote_path=''):
 
     # Copies the file to its final destination in the container
     target.ContainerCopy(file_name, remote_path)
+
+
+class JujuMixin(DebianMixin):
+  """
+  Class to allow running juju-deployed workloads
+
+  Bootstraps a Juju environment using the manual provider:
+  https://jujucharms.com/docs/stable/config-manual
+  """
+
+  OS_TYPE = os_types.JUJU
+
+  isController = False
+
+  """
```

Comment:
Please switch to comments over multiline strings (except in the case of docstrings)